embedded-google-storage/README.adoc

```diff
@@ -0,0 +1,60 @@
+=== embedded-google-storage
+
+==== Maven dependency
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+    <groupId>com.playtika.testcontainers</groupId>
+    <artifactId>embedded-google-storage</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+==== Consumes (via `bootstrap.properties`)
+
+* `embedded.google.storage.enabled` `(true|false, default is true)`
+* `embedded.google.storage.reuseContainer` `(true|false, default is false)`
+* `embedded.google.storage.dockerImage` `(default is 'fsouza/fake-gcs-server')`
+** Image versions on https://hub.docker.com/r/fsouza/fake-gcs-server/tags[dockerhub]
+* `embedded.google.storage.projectId` `(project id for storage, default is my-project-id)`
+* `embedded.google.storage.bucketLocation` `(location for buckets, default is US-CENTRAL1)`
+
+* Predefined buckets:
```

Comment:
Can you please rephrase this part to smth like:
`embedded.google.storage.buckets` creates buckets on startup